app/screens/home/home_screen.tsx

```diff
@@ -22,7 +23,7 @@ export default function HomeScreen({ user, navigation, firestoreCtrl }: any) {
       const fetchChallenges = async () => {
         try {
           const challengesData = await firestoreCtrl.getChallengesByUserId(uid);
-          console.log("Challenges", challengesData);
+          console.log("Challenges [" + uid + "]", challengesData);
           setChallenges(challengesData);
         } catch (error) {
```

Comment:
Maybe you should add a toast or an alert for the user in this case to improve user experience.

Reply:
I'm not sure to understand what you mean there, those are raw data to help debug, before displaying in a nicer way that data to the user.